kong/db/strategies/cassandra/init.lua

```diff
@@ -575,9 +561,19 @@ function _mt:insert(entity, options)
       end
 
       insert(args, serialize_arg(field, entity[field_name]))
+      insert(cols, field_name)
+      insert(binds, "?")
     end
+
+    ::continue::
   end
 
+  local cql = fmt("INSERT INTO %s(%s) VALUES(%s)%s",
+                  schema.name,
+                  concat(cols, ", "),
+                  concat(binds, ", "),
+                  ttl and fmt(" USING TTL %d", ttl) or "")
```

Comment:
This function suddenly became more costly for all Cassandra versions. Was there no way to bind [cassandra.unset](https://thibaultcha.github.io/lua-cassandra/modules/cassandra.html#type_serializers) instead?

Reply:
I haven't tried that approach, I just went with copying the old DAO approach which worked on C* 2.2 — giving cassandra.unset a spin now

Reply:
It seems like this moved to query-build-time anyway